<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateFilm</class>
 <widget class="QDialog" name="CreateFilm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>465</width>
    <height>544</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cr√©er un film</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="1">
      <layout class="QFormLayout" name="formLayout">
       <item row="2" column="0" colspan="2">
        <widget class="QPushButton" name="pushButton_2">
         <property name="styleSheet">
          <string notr="true">margin-bottom: 10px;
padding: 10px;</string>
         </property>
         <property name="text">
          <string>Selectionner une image</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="titreLabel">
         <property name="text">
          <string>Titre</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0" colspan="2">
        <widget class="QLineEdit" name="titreLineEdit"/>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="descriptionLabel">
         <property name="text">
          <string>Description</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0" colspan="2">
        <widget class="QLineEdit" name="descriptionLineEdit"/>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="realisateurLabel">
         <property name="text">
          <string>Realisateur</string>
         </property>
        </widget>
       </item>
       <item row="8" column="0" colspan="2">
        <widget class="QLineEdit" name="realisateurLineEdit"/>
       </item>
       <item row="9" column="0">
        <widget class="QLabel" name="dureeLabel">
         <property name="text">
          <string>Duree</string>
         </property>
        </widget>
       </item>
       <item row="10" column="0" colspan="2">
        <widget class="QLineEdit" name="dureeLineEdit"/>
       </item>
       <item row="11" column="0">
        <widget class="QLabel" name="genreLabel">
         <property name="text">
          <string>Genre</string>
         </property>
        </widget>
       </item>
       <item row="12" column="0" colspan="2">
        <widget class="QLineEdit" name="genreLineEdit"/>
       </item>
       <item row="13" column="0">
        <widget class="QLabel" name="dateDeSortieLabel">
         <property name="text">
          <string>Date de sortie</string>
         </property>
        </widget>
       </item>
       <item row="14" column="0" colspan="2">
        <widget class="QDateEdit" name="dateDeSortieDateEdit"/>
       </item>
       <item row="15" column="0">
        <widget class="QLabel" name="prixLabel">
         <property name="text">
          <string>Prix</string>
         </property>
        </widget>
       </item>
       <item row="16" column="0" colspan="2">
        <widget class="QLineEdit" name="prixLineEdit"/>
       </item>
       <item row="17" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>10</width>
           <height>10</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="18" column="0" colspan="2">
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>Creer</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QLabel" name="error">
         <property name="styleSheet">
          <string notr="true">font-size: 20px;
color: &quot;#FCA311&quot;;
margin-bottom: 16px;</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="3">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Maximum</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>60</width>
         <height>200</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="0">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Maximum</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>60</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
